lib/pcre.mli

```diff
@@ -29,6 +29,12 @@ val exec : rex:regexp -> ?pos:int -> string -> groups
 val get_substring : groups -> int -> string
 (** Equivalent to {!Core.Group.get}. *)
 
+val names : regexp -> string array
+(** Return the names of named groups. *)
+
+val get_named_substring : regexp -> string -> groups -> string
+(** Return the matched named group, or raise [Not_found]. *)
```

Comment:
The doc-comment should  should mention the behavior in case of duplicated names

Reply:
Done. And I slightly changed the behavior, for (what I think is) a more useful one.